### PR TITLE
Change footer design

### LINKED ISSUE #316: Update the footer design
Update the footer to look like the following, 
The following image is an example, no need to implement the exact same. (ex: no need for rounded icons. We can use current icons)

Screenshot 2022-07-31 at 21 46 37

Tasks:

- [ ] Change bg colour to #333333
- [ ] Change the copyright statement as provided in the image
- [ ] Linkt "Our Legal Status in Sri Lanka" to https://www.redcross.lk/who-we-are/our-legal-status-in-sri-lanka/
- [ ] All links should open in a new tab

### PR DESCRIPTION
**Pull request by RandilaP**
## Purpose

The purpose of this PR is to fix #316 

## Goals
-Update footer UI

### Screenshots
image

### CODE REVIEW

File: donor-portal/src/components/footer/styles.css [hunk 1]
```diff
@@ -4,20 +4,13 @@
...
 
-.footer p {
-  color: white;
-  font-size: 18px;
-  width: 30%;
-  margin: auto;
-  font-weight: 600;
-  margin-bottom: 15px;
-}
+
```
Comment (anjula-sack) [inline donor-portal/src/components/footer/styles.css:7]: Remove this empty line

File: donor-portal/src/components/footer/styles.css [hunk 2]
```diff
@@ -76,55 +72,52 @@
...
+  margin: auto auto 35px auto;
+  
+}
+
+.footer-gray  a {
+  text-decoration: underline;
+  color: #c3c3c3;
+  
+ 
+
```
Comment (anjula-sack) [inline donor-portal/src/components/footer/styles.css:120]: Remove this empty lines

File: donor-portal/src/components/footer/styles.css [hunk 3]
```diff
@@ -76,55 +72,52 @@
...
-  padding: 0 25px;
-  transition: all 0.5s;
+  padding: 0 10px;
+  
 }
 
-a.text-link:hover {
-  padding: 0 60px;
-}
+
```
Comment (anjula-sack) [inline donor-portal/src/components/footer/styles.css:93]: same here

File: donor-portal/src/components/footer/styles.css [hunk 4]
```diff
@@ -76,55 +72,52 @@
...
 .footer-black-bottom span {
   color: white;
 }
 
 .text-link {
   color: white;
-  padding: 0 25px;
-  transition: all 0.5s;
+  padding: 0 10px;
+
```
Comment (anjula-sack) [inline donor-portal/src/components/footer/styles.css:90]: Remove the empty line

File: donor-portal/src/components/footer/styles.css [hunk 5]
```diff
@@ -76,55 +72,52 @@
   display: flex;
   justify-content: center;
   text-decoration: none;
-  font-size: 18px;
+  font-size: 0.8125rem;
   margin: auto;
   width: 65%;
-  font-weight: 600;
+
```
Comment (anjula-sack) [inline donor-portal/src/components/footer/styles.css:78]: same here

File: donor-portal/src/components/footer/styles.css [hunk 6]
```diff
@@ -76,55 +72,52 @@
...
   margin: auto;
   width: 65%;
-  font-weight: 600;
+  
 }
 
-.footer-black-bottom:hover a {
-  padding: 0 18px;
-}
+
```
Comment (anjula-sack) [inline donor-portal/src/components/footer/styles.css:81]: same here

File: donor-portal/src/components/footer/index.tsx [hunk 7]
```diff
@@ -48,8 +43,12 @@ export default function Footer() {
       </div>
       <div className="footer-gray">
         <p>
-          Copyright © <a href="www.elixir.redcross.lk">elixir.redcross.lk </a>|
-          All rights reserved
+          All content copyright © 2022 Sri Lanka Red Cross. All rights reserved.{" "}
+          |
+          <a href=" https://www.redcross.lk/who-we-are/our-legal-status-in-sri-lanka/">
+            {" "}
+            Our Legal Status in Sri Lanka{" "}
```
Comment (anjula-sack) [inline donor-portal/src/components/footer/index.tsx:50]:
```suggestion
            Our Legal Status in Sri Lanka
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done @RandilaP !

**Pull request merged**